src/xia2/cli/cluster_analysis.py

```diff
@@ -0,0 +1,284 @@
+                identifiers_overall_cluster = [
+                    MCA._data_manager.ids_to_identifiers_map[l]
+                    for l in overall_cluster.labels
+                ]
+                identifiers_to_exclude = [
+                    MCA._data_manager.ids_to_identifiers_map[l] for l in cluster.labels
+                ]
+                identifiers_to_output = [
+                    i
+                    for i in identifiers_overall_cluster
+                    if i not in identifiers_to_exclude
+                ]
+                output_cluster(
+                    new_folder, cluster, MCA._data_manager, identifiers_to_output
+                )
+
+        for cluster in MCA._cluster_analysis.cos_angle_clusters:
+            if (
+                "cluster_" + str(cluster.cluster_id) in cos_list
+                or cluster.cluster_id == params.output_cos_cluster_number
+            ):
+                new_folder = (
+                    "cos_angle_clusters/" + "cluster_" + str(cluster.cluster_id)
+                )
+                output_cluster(
+                    new_folder, cluster, MCA._data_manager, cluster_identifiers
+                )
+
+            if params.exclude_cos_cluster_number == cluster.cluster_id:
+                new_folder = (
+                    "cos_angle_clusters/"
+                    + "excluded_cluster_"
+                    + str(cluster.cluster_id)
+                )
+                overall_cluster = MCA._cluster_analysis.cos_angle_clusters[-1]
+                identifiers_overall_cluster = [
+                    MCA._data_manager.ids_to_identifiers_map[l]
+                    for l in overall_cluster.labels
+                ]
+                identifiers_to_exclude = [
+                    MCA._data_manager.ids_to_identifiers_map[l] for l in cluster.labels
+                ]
+                identifiers_to_output = [
+                    i
+                    for i in identifiers_overall_cluster
+                    if i not in identifiers_to_exclude
+                ]
+                output_cluster(
+                    new_folder, cluster, MCA._data_manager, identifiers_to_output
+                )
+
+        id_list = []
+        table_list = [["Experiment/Image Number", "Image Template"]]
+
+        el = MCA._data_manager.experiments
+        ids = list(el.identifiers())
+        for item in el:
+            id = MCA._data_manager.identifiers_to_ids_map[item.identifier]
+            id_list.append(id)
+
+        for j, item in enumerate(ids):
+            expt = el[ids.index(item)]
+            i = expt.imageset
+            table_list.append([id_list[j], i.paths()[0]])
+
+        loader = ChoiceLoader(
+            [PackageLoader("xia2", "templates"), PackageLoader("dials", "templates")]
+        )
+        env = Environment(loader=loader)
+
+        template = env.get_template("clusters.html")
+        html = template.render(
+            page_title=params.title,
```

Comment:
```suggestion
            page_title="xia2 cluster analysis",
```
If we set the title here, then we can remove a few phil options.